Remove the defaults from the style wrapper injection. Remove the mean… (#3794)
plone/volto: docs/source/blocks/block-style-wrapper.md

## Enabling Style Wrapper in a block

The wrapper is always present in the rendering of both the view and edit components.
Once you add the `styles` field in your block schema, the wrapper will inject the class names derived from it into both the view and the edit components.

### Using `styles` field in your block

The wrapper builds the class names to be injected by looking up a field called `styles` in your block schema.

> As a schema it has this signature, and it's normally placed in a `Styling` fieldset:

Reviewer: ```suggestion
As a schema it has the following signature, and it's normally placed in a `Styling` fieldset:
```

```js
const stylingSchema = {
  fieldsets: [{
    id: 'styling',
    title: intl.formatMessage(messages.styling),
    fields: ['styles'],
  }],

  properties: {
    styles: {
      widget: 'object',
      title: intl.formatMessage(messages.styling),
      schema: stylesSchema({

>         schema: defaultStyleSchema({ formData, intl }),

Reviewer 2: @sneridagh this code snippet that is still added references **defaultStyleSchema**
which is now removed from this pull request.

Shouldn't the code snippet explain the empty styling fieldset from
https://github.com/plone/volto/pull/3794/files?short_path=43e361f#diff-8f34daa1151a8f71198bb80a59f0af841606e03e8e04c1c30a9825527283b279R296
which is then populated from 
https://github.com/plone/volto/pull/3794/files#diff-43e361f4bd5a52502dd6b2af742c7fb403abc31e7e8a61be75d76a6c49a733bfR70
?

Author: Fixed.

        formData,
        intl,
      }),
    }
  }
}
```

The `addStyling` helper adds the fieldset and the `styles` field inside the `Styling` fieldset for you, so when defining your block schema you can do:

```js
import { addStyling } from '@plone/volto/helpers/Extensions/withBlockSchemaEnhancer';

export const TeaserSchema = ({ intl }) => {
  const schema = {
  // Here your block schema
  };

  addStyling({ schema, intl });

  // Here you add your custom styling properties to the `styles` object
  schema.properties.styles.schema.properties.align = {
    widget: 'align',
    title: intl.formatMessage(messages.align),
    actions: ['left', 'right', 'center'],
  };

  // and finally add it to the default fieldset
  schema.properties.styles.schema.fieldsets[0].fields.push('align');

  return schema;
};
```

> You can add a set of style fields defining your block styles (eg. alignment, background color, etc...) in your block schema by adding them to the `styles` object field as shown above.

Reviewer: ```suggestion
You can add a set of style fields defining your block styles—such as alignment, background color, and so on—in your block schema by adding them to the `styles` object field as shown above.
```

## The `styles` field

The `styles` field is mapped to an `objectWidget`.

> This is the shape of a possible set of styles.

Reviewer: ```suggestion
The following is an example of a possible set of styles.
```

> The Style Wrapper will read them and inject the styles in the edit and view components as is shown in the next sections.

Reviewer: I'm not sure why "Style Wrapper" is capitalized. It's not a proper or formal noun, is it?

```suggestion
The style wrapper will read the styles and inject them into the edit and view components as shown in the next sections.
```

Then it's at your discretion how you define the CSS class names in your theme.

## Align class injection

> There is an automatic class name injection happening at the same time the Style Wrapper class names injection.

Reviewer: Is capitalization required?

```suggestion
There is an automatic class name injection happening at the same time the style wrapper class names injection.
```

The `data.align` is also injected directly.
This is in place to help properly position the block in the current layout and play well with legacy CSS and block layout.

> This might be replaced in the future by the Style Wrapper class names injection.

Reviewer: ```suggestion
This might be replaced in the future by the style wrapper class names injection.
```
